config/manifests/bases/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -73,7 +73,7 @@ spec:
 
       On the Install Operator page:
 
-      - Select `stable-1.2` from the list of available Update Channel options.
+      - Select `stable-1.3` from the list of available Update Channel options.
```

Comment:
For upstream master/main branch we are using `candidate` channel.  For downstream (from release branches) we'll be using `stable-1.x` channel

Reply:
changed it to say use `candidate` for latest and `stable-x.y` for already release versions